docs/platform/release-notes/tizen-6-5-m2.md

```diff
@@ -0,0 +1,400 @@
+# Tizen 6.5 Public M2
+
+Release date: Oct. 31, 2021
+
+
+## Release Notes
+
+### System (Kernel and System framework)
+
+#### New and changed features
+
+- New API and features
+  - Kernel
+    - A finegrained version of THP (Transparent Hugepage) in Linux kernel for embedded systems has been implemented in RPI4.
+  - HAL
+    - HAL API Layer is implemented for unifying the HAL backend style and splitting out the hardware dependency.
+    - New image structure with hal.img is adapted for HAL, a set of HAL backend set in /hal.
+  - System and resource management
+    - VIP process monitor has been improved by removing race conditions.
+    - Percentage based memory threshold has been developed.
+    - Coredump generation for .NET 6 runtime has been supported.
+    - Tizen feature database tools have been improved.
+    - Container adaptation for file system mount units has been developed.
+  - Device and sensor management
+    - Orientation and rotation vector sensor algorithms have been upgraded for improving correctness.
+    - HAL backend for inertial measurement unit (IMU) sensors has been implemented for RPI4 devices.
+    - Separate backend libraries for sensor devices have been supported.
+    - Daemon-less direct API set for peripheral-io and runtime memory usage has been developed.
+    - Sleep mode for IoT headless devices has been supported.
+    - Display control for HDMI-based external display devices has been supported.
+    - Display option for USB connection change has been added.
+    - API for board serial number has been added.
+  - OS upgrade
+    - FOTA update for RPI4 IoT headless devices has been supported.

```

Comment:
Global comment recommend to change
RPI4 => Raspberry Pi 4

Reply:
I accept this.